js/client/modules/@arangodb/testsuites/gtest.js

```diff
@@ -112,19 +112,33 @@ function getGTestResults(fileName, defaultResults) {
   return results;
 }
 
-function gtestRunner (testname, options) {
+function gtestRunner (testfilename, name, opts, testoptions) {
+  let options = _.clone(opts);
```

Comment:
Bettre to use `cloneDeep()` ?

Reply:
options is one-dimensional (see testing.js)